model/websocket_request.go

```diff
@@ -20,6 +21,7 @@ type WebSocketRequest struct {
 	Session Session            `json:"-" msgpack:"-"`
 	T       i18n.TranslateFunc `json:"-" msgpack:"-"`
 	Locale  string             `json:"-" msgpack:"-"`
+	Logger  mlog.LoggerIFace   `json:"-" msgpack:"-"`
```

Comment:
Hmm .. wondering why do we need the logger as part of this struct?

Reply:
Oh, that one shouldn't be in this PR. It was part of an abandoned approach. Very sorry for this.